Attendance_app.py

```diff
@@ -192,9 +197,21 @@ def __init__(self, parent, controller):
             font=("Courier", text_fontsize),
         )
         self.lb_class.place(x=170 * scale_factor, y=155 * scale_factor)
-        self.tv_class = tk.Entry(self, width=entry_width)
-        self.tv_class.focus()
-        self.tv_class.place(x=340 * scale_factor, y=160 * scale_factor)
+
+        # Class Codes Autocompletion Stuff
+        self.filled = False
+
+        self.classCodes = list(class_codes)
+        self.class_variable = StringVar(self)
+        self.class_variable.set("")  # default value
+        self.tv_class = tk.Entry(self, textvariable=self.class_variable)
+
+        self.tv_class.focus_set()
+        self.tv_class.bind("<KeyRelease>", self.get_typed)
+        self.tv_class.bind("<Key>", self.filled)
+        self.tv_class.icursor("end")
+
+        self.tv_class.place(x=400 * scale_factor, y=160 * scale_factor)

```

Comment:
⚠️ `x` position been changed from `340` to `400`.
Rationale (behind this change)?